comid/measurement.go

```diff
@@ -728,10 +771,50 @@ func (o Measurement) Valid() error {
 		}
 	}
 
-	// check non-empty<> condition on measurement-values-map
-	if err := o.Val.Valid(); err != nil {
-		return err
-	}
+	return o.Val.Valid()
+}
 
-	return nil
+// Measurements is a container for Measurement instances and their extensions.
```

Comment:
The whole Measurements was `in Measurements.go` file, any reason why this is moved here ?

The file already is very long...?

Reply:
It's under 1000 lines, that's hardly "long" :). 

The implementation of `Measurements` is trivial, and it doesn't seem to make much sense spliting that out when the rest of measurement-related stuff is in here. Having things spread a cross a multitude of files, each containing one struct just makes things more annoying to manage.